data/assets/educational/scale/eiffeltower.asset

```diff
@@ -0,0 +1,84 @@
+local earthAsset = asset.require('scene/solarsystem/planets/earth/earth')
+local sunAsset = asset.require('scene/solarsystem/sun/sun')
+
+local modelFolder = asset.syncedResource({
+	Name = "Eiffel Tower Model",
+	Type = "HttpSynchronization",
+	Identifier = "eiffel_tower_model",
+	Version = 1
+  })
+  
+local eiffelTower = {
+	Identifier = "eiffelTower",
+	Parent = earthAsset.Earth.Identifier,
+	Renderable = {
+		Type = "RenderableModel",
+		GeometryFile = modelFolder .. "eiffeltower.osmodel",
+		ModelScale = "Centimeter",
+		RotationVector = {0,45,0},
+		LightSources = {
+			{
+				Type = "SceneGraphLightSource",
+				Identifier = "sunLightSourceForEiffelTower",
+				Node = sunAsset.Sun.Identifier
+			},
+		}
+	},
+	GUI = {
+		Name = "Eiffel Tower",
+		Path = "/Scale Objects"
+	},
+	Transform = {
```

Comment:
We usually specify the Translation classes before the Renderable and I think it would be good to do that here, too